docs/coordinates/frames.rst

```diff
@@ -457,3 +575,95 @@ process.
 For a demonstration of how to define transformation functions that also work for
 transforming velocity components, see
 :ref:`astropy-coordinate-transform-with-velocities`.
+
+
+Supporting velocity data in frames
+----------------------------------
+
+As alluded to by point (4) in the introduction above
+:ref:`astropy-coordinates-design`, the examples we have seen above mostly deal
+with customizing frame behavior for positional information. (For some context
+about how velocities are handled in ``astropy.coordinates``, it may be useful to
+read the overview: :ref:`astropy-coordinate-custom-frame-with-velocities`.)
+
+When defining a frame class, it is also possible to set a
+``default_differential`` (analogous to ``default_representation``), and to
+customize how velocity data components are named. Expanding on our custom frame
+example above, we can use `~astropy.coordinates.RepresentationMapping` to
+override ``Differential`` component names. The default ``Differential``
+components are typically named after the corresponding ``Representation``
+component, preceded by ``d_``. So, for example, the longitude ``Diffferential``
+component is, by default, ``d_lon``. However, there are some defaults to be aware of. Here, if we set the
+default ``Differential`` class to also be Spherical, it will implement a set of
+default "nicer" names for the velocity components, mapping ``pm_R`` to
+``d_lon``, ``pm_D`` to ``d_lat``, and ``radial_velocity`` to ``d_distance``
```

Comment:
Probably good to add the `pm_R_cosD` version?

Reply:
You mean a new example frame here with `SphericalCosLatDifferential`? As this example is mainly about overriding component names, I'm not sure it is necessary (we link to the "Creating Frame Objects with Velocity Data" page that has more detail about the differential classes themselves). But maybe I misunderstood your comment!

Reply:
No, I just meant to mention `pm_R_cosD` as one of the standard "nicer" names (and I think I'd use it too, instead of `pm_R`, but that is less important).